sklearn/linear_model/tests/test_logistic.py

```diff
@@ -74,17 +74,14 @@ def test_error():
     assert_raises_regexp(ValueError, msg,
                          LogisticRegression(C="test").fit, X, Y1)
 
-    msg = "Tolerance for stopping criteria must be positive"
-    assert_raises_regexp(ValueError, msg,
-                         LogisticRegression(tol=-1).fit, X, Y1)
-    assert_raises_regexp(ValueError, msg,
-                         LogisticRegression(tol="test").fit, X, Y1)
+    for LR in [LogisticRegression, LogisticRegressionCV]:
+        msg = "Tolerance for stopping criteria must be positive"
+        assert_raises_regexp(ValueError, msg, LR(tol=-1).fit, X, Y1)
+        assert_raises_regexp(ValueError, msg, LR(tol="test").fit, X, Y1)
```

Comment:
The failure message is easier to read with `assert_raise_message` than `assert_raises_regexp` when regexp expression are not required.